lib/browser/api/browser-window.js

```diff
@@ -96,17 +95,6 @@ BrowserWindow.prototype._init = function () {
   // Notify the creation of the window.
   app.emit('browser-window-created', {}, this)
 
-  // Be compatible with old APIs.
```

Comment:
Yeah, I think it is still valuable cleanup to do before 1.0 or else we'd leave them in until 2.0.